### DISCUSSION TITLE
Table Borders - style not pulling correctly in PDF

### DISCUSSION THREAD

**Discussion by kobrien603** [spatie/browsershot#569] [Bugs]
I have a dynamically built table with a border radius on all corners of the table. When I push it to a PDF the corners start to fade away.

Has anyone else experienced this? I have not been able to find any online solutions. I have tried increasing the border size but then the borderlines do not match up properly.

Any suggestions would be greatly appriciated

PHP 7.3 Browsershot V.9.1.1

Browsershot::html($pdf)
            ->noSandbox()
            ->showBackground()
            ->ignoreHttpsErrors()
            ->emulateMedia("screen")
            ->waitUntilNetworkIdle(false)
            ->save($file_path);

![Capture](https://user-images.githubusercontent.com/47003518/135117913-092cd2a0-83bf-48f5-ad5b-33682c908618.PNG)

**Comment by kobrien603** [comment]
Here is a better picture!

![image](https://user-images.githubusercontent.com/47003518/135122200-88c859ad-8a8f-4d37-8f59-b5052b57f054.png)

**Reply by jasperfernandez** [reply]
Experiencing this too
![image](https://github.com/user-attachments/assets/8e6a0dde-56eb-457b-8529-97042278ac79)
